pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -351,7 +351,7 @@
     "hitron_coda" = ps: with ps; [ ];
     "hive" = ps: with ps; [ ]; # missing inputs: pyhiveapi
     "hlk_sw16" = ps: with ps; [ ]; # missing inputs: hlk-sw16
-    "home_connect" = ps: with ps; [ aiohttp-cors ]; # missing inputs: homeconnect
+    "home_connect" = ps: with ps; [ aiohttp-cors homeconnect ];
```

Comment:
Can we enable tests for any of them?

Reply:
The `home_connect` has tests. It's worth a try, especially as they are already present for a while.